Samples/All/BumbleRumble/Common/NetworkManager.cpp

```diff
@@ -1224,3 +1199,74 @@ PartyChatControl* NetworkManager::GetChatControl(std::string& peer)
     }
     return nullptr;
 }
+
+void NetworkManager::SetCognitiveServicesEnabled(bool bEnabled)
+{
+	m_enableCognitiveServices = bEnabled;
+
+	// For the purposes of the sample, enable or disable all possible features
+	SetTextChatTranslationOptions(bEnabled);
+	SetVoiceChatTranscriptionOptions(bEnabled, bEnabled, bEnabled, false, bEnabled); 
+}
+
+void NetworkManager::SetTextChatTranslationOptions(bool bTranslateToLocalLanguage)
+{
+    DEBUGLOG("SetTextChatTranslationOptions: bTranslateToLocalLanguage = %s", bTranslateToLocalLanguage ? "Enabled" : "Disabled");
+
+    // Enable translation to local language in chat controls.
+    PartyError err = m_localChatControl->SetTextChatOptions(
+        bTranslateToLocalLanguage ? PartyTextChatOptions::TranslateToLocalLanguage : PartyTextChatOptions::None,
+        nullptr
+    );
+
+    if (PARTY_FAILED(err))
+    {
+        DEBUGLOG("SetTextChatOptions failed: %s\n", GetErrorMessage(err));
+    }
+}
+
+void NetworkManager::SetVoiceChatTranscriptionOptions(bool bTranscribeSelf, bool bTranscribeOtherChatControlsWithMatchingLanguages, bool bTranscribeOtherChatControlsWithNonMatchingLanguages, bool bDisableHypothesisPhrases, bool bTranslateToLocalLanguage)
+{
+    if (m_localChatControl)
+    {
+        DEBUGLOG("Setting Speech-To-Text options.\n");
+
+        PartyVoiceChatTranscriptionOptions Options = PartyVoiceChatTranscriptionOptions::None;
+
+        if (bTranscribeSelf)
+        {
+            Options |= PartyVoiceChatTranscriptionOptions::TranscribeSelf;
+        }
+
+        if (bTranscribeOtherChatControlsWithMatchingLanguages)
+        {
+            Options |= PartyVoiceChatTranscriptionOptions::TranscribeOtherChatControlsWithMatchingLanguages;
+        }
+
+        if (bTranscribeOtherChatControlsWithNonMatchingLanguages)
+        {
+            Options |= PartyVoiceChatTranscriptionOptions::TranscribeOtherChatControlsWithNonMatchingLanguages;
+        }
+
+        if (bDisableHypothesisPhrases)
+        {
+            Options |= PartyVoiceChatTranscriptionOptions::DisableHypothesisPhrases;
+        }

```

Comment:
Please remove anything that has to do with DisableHypothesisPhrases. It isn't implemented.